layers/meta-balena-beaglebone/recipes-core/images/balena-image.inc

```diff
@@ -25,6 +30,24 @@ BALENA_BOOT_PARTITION_FILES:beagleboard-xm = " \
     u-boot.img: \
 "
 
+BALENA_BOOT_PARTITION_FILES:beagleplay = " \
+    tispl.bin_unsigned:/tispl.bin \
+    u-boot.img: \
+    tiboot3.bin: \
+    ${KERNEL_IMAGETYPE}${KERNEL_INITRAMFS}-${MACHINE}.bin:/${KERNEL_IMAGETYPE} \
+    k3-am625-beagleplay.dtb: \
+    extlinux.conf:/extlinux/extlinux.conf \
+"
+
+BALENA_BOOT_PARTITION_FILES:beagleplay-k3r5 = " \
```

Comment:
Why do we need to copy these files for the k3r5 machine? We are not using that, are we?

Reply:
I think the boot sequence involves loading the spl onto the k3r5, but we not have to specifically create a boot partition for it. Will remove this block and retest.

Reply:
Done.